src/components/10-atoms/link/story.js

```diff
@@ -63,4 +63,11 @@ storiesOf('Atoms/Link', module)
       <axa-link href="https://axa.ch/en/private-customers.html" variant="arrowleft-animated-white">Motion Link Left</axa-link><br><br>
       <axa-link href="https://axa.ch/en/private-customers.html" variant="arrowright-animated-white">Motion Link Right</axa-link>
       `
+  )
+  .add(
```

Comment:
All the other stories have only one variant if I remember correctly, we do we have two in this story? I'm not sure which way we should go, but we should at least be consistent imho :)

Reply:
i dont get this one what u mean